### ISSUE TITLE
Using `/` as the URL doesn't work as expected.

### ISSUE THREAD

**Issue by RemiKalbe** [VirtusLab-Open-Source/strapi-plugin-navigation#366]
Let's say I have some page where I want it to be available at `/`.

![CleanShot 2023-08-09 at 16 22 04@2x](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/8604600/dd5815d1-65a0-415f-89ef-e7af640e398b)

The output path show `/` which is expected, then I try to query my navigation using GraphQL:

```graphql
query {
  renderNavigation(
    navigationIdOrSlug: "main-navigation"
    locale: "fr"
    path: "/"
  ) {
    id
    title
    path
    related {
      id
      attributes {
        __typename
        ... on Page {
          Title
        }
      }
    }
  }
}
```

But it returns nothing

<img width="757" alt="CleanShot 2023-08-09 at 16 24 34@2x" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/8604600/ee24d1ae-93cf-483b-b279-f5236dd4b5d8">

After some investigation I found that I actually have to write `path: "//"` instead of `path: "/"`

<img width="871" alt="CleanShot 2023-08-09 at 16 25 20@2x" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/8604600/be8973af-f8cf-48f5-91c0-e643e61ea165">

It doesn't end there! Let's add a nested route to it

<img width="814" alt="CleanShot 2023-08-09 at 16 26 02@2x" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/8604600/49e75506-3a57-4dd1-8502-a6f79c6e414a">

As you can see the expected path is `/nested`, but again, it doesn't work, the actual path that work is `///nested`

<img width="816" alt="CleanShot 2023-08-09 at 16 26 39@2x" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/8604600/24838e94-fca6-423a-94b4-9746a435648c">

**Comment by RemiKalbe** [comment]
After looking at the codebase I think the fix would be to allow having an empty URL if it's the root?

**Comment by cyp3rius** [comment]
Fixed in version `2.2.14`
